MovieJournalMVC/View/MovieReviewCell.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  MovieReviewCell.swift
+//  MovieJournalMVC
+//
+//  Created by Lauren Small on 3/12/19.
+//  Copyright © 2019 Lauren Small. All rights reserved.
+//
+
+import UIKit
+
+class MovieReviewCell: UITableViewCell {
+    
+    @IBOutlet weak var movieTitleLabel: UILabel!
+    
+    var title: String = ""
```

Comment:
Again, setting meaningless default values like this could get you into trouble in the future, as you have no way to detect you didn't set things up correctly until the user sees it. Better to pass them into a setup function of some kind so you can set the label.

Reply:
Default values are gone 👍